src/transformers/models/auto/tokenization_auto.py

```diff
@@ -456,7 +456,13 @@
                     "SeamlessM4TTokenizerFast" if is_tokenizers_available() else None,
                 ),
             ),
-            ("siglip", ("SiglipTokenizer" if is_sentencepiece_available() else None, None)),
+            (
+                "siglip",
+                (
+                    "SiglipTokenizer" if is_sentencepiece_available() else None,
+                    "PreTrainedTokenizerFast" if is_tokenizers_available() else None,
+                ),
+            ),
```

Comment:
ref for https://github.com/huggingface/transformers/pull/33751#issuecomment-2395040571